utils.py

```diff
@@ -91,4 +104,232 @@ def generate_credentials():
     my_cred_file = loader.load("google_secret_format.txt")
     result = my_cred_file.generate().decode("utf-8") % (conf.GOOGLE_CLIENT_ID, conf.GOOGLE_PROJECT_ID, conf.GOOGLE_CLIENT_SECRET, conf.BASE_URL+"/docs/")
     with open(conf.CLIENT_SECRETS_FILE, "w") as cred_json:
-        cred_json.write(result)
\ No newline at end of file
+        cred_json.write(result)
```

Comment:
Should this be in pip or some package manager?

Reply:
Sii! Es loque estabamos platicando seria mejor hacer una mini libreria de esto jeje